sdks/java/io/hdfs/pom.xml

```diff
@@ -58,12 +58,41 @@
     </dependency>
 
     <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-annotations</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.code.findbugs</groupId>
+      <artifactId>jsr305</artifactId>
+    </dependency>
+
+    <dependency>
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-client</artifactId>
       <version>2.7.0</version>
       <scope>provided</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-common</artifactId>
+      <version>2.7.0</version>
```

Comment:
Spark can be built against a wide range of hadoops version, as stated [here](http://spark.apache.org/docs/latest/building-spark.html#specifying-the-hadoop-version)
Two questions: 
1. Why does HadoopIO depends specifically on 2.7.0 ? 
2. Would we want to provide different hadoop.version profiles for Beam as well ?